enterprise/metrics/src/main/java/org/neo4j/metrics/output/OutputBuilder.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.metrics.output;
+
+import com.codahale.metrics.MetricRegistry;
+
+import org.neo4j.cluster.ClusterSettings;
+import org.neo4j.helpers.HostnamePort;
+import org.neo4j.kernel.configuration.Config;
+import org.neo4j.kernel.impl.spi.KernelContext;
+import org.neo4j.kernel.lifecycle.LifeSupport;
+import org.neo4j.logging.Log;
+import org.neo4j.metrics.MetricsSettings;
+
+import static org.neo4j.metrics.MetricsSettings.csvEnabled;
+import static org.neo4j.metrics.MetricsSettings.gangliaEnabled;
+import static org.neo4j.metrics.MetricsSettings.gangliaInterval;
+import static org.neo4j.metrics.MetricsSettings.gangliaServer;
+import static org.neo4j.metrics.MetricsSettings.graphiteEnabled;
+import static org.neo4j.metrics.MetricsSettings.graphiteInterval;
+import static org.neo4j.metrics.MetricsSettings.graphiteServer;
+
+public class OutputBuilder
+{
+    private Config config;
+    private MetricRegistry registry;
+    private Log logger;
+    private KernelContext kernelContext;
+    private LifeSupport life;
+
+    public OutputBuilder( Config config, MetricRegistry registry, Log logger, KernelContext kernelContext,
+            LifeSupport life )
+    {
+        this.config = config;
+        this.registry = registry;
+        this.logger = logger;
+        this.kernelContext = kernelContext;
+        this.life = life;
+    }
+
+    public EventReporter build()
```

Comment:
A bit of java doc should be helpful here, look strange that we calling build on OutputBuilder and receiving something else (not Output...) as result, maybe rename OutputBuilder to more descriptive name for example: EventReporterBuilder as example?